### ISSUE TITLE
Unsatisfying opam behavior with the README instructions

### ISSUE THREAD

**Issue by gasche** [ocaml-multicore/multicore-opam#15]
At the time of writing, the README instructions for trying out Multicore-OCaml are as follows:

```
opam repository add multicore https://github.com/ocamllabs/multicore-opam.git --all
opam update
opam switch create 4.06.1+multicore --empty
opam switch 4.06.1+multicore
opam install ocaml-variants.4.06.1+multicore
```

I have observed the following usability issues on my machine (opam 2.0.4):

1. The multicore repository is added to all existing switches, but it is not marked as a "default switch" so it is not visible from the new switch.
2. The --empty dance has the result that the OCaml version is not fixed for the switch. In particular, `opam install merlin` immediately suggests to update ocaml-variants from 4.06.0+multicore to 4.10.0.

To solve issue (1), I propose to define and enable the multicore switch at switch-creation time. (This is less invasive than adding it to all switches or to default switches; I would prefer development repositories to not touch my normal switches at all.)

To solve issue (2), I propose to not use `--empty`.

The instructions thus become as follows:

```
opam update
opam switch create 4.06.1+multicore --repositories=default,multicore=git+https://github.com/ocamllabs/multicore-opam.git
```

Note: the semantics of adding a new repository at switch-creation time let us use a repository name that already exists, provided the URL is exactly the same to the one opam remembers. In addition, it seems that when we provide a https://foo.git repo URL, opam actually remembers git+https://foo.git.  For this reason, it is important (with my opam version) to use git+https in the URL above -- if you followed the "old" procedure and have a multicore repo already, the instruction above will work with git+https but would fail with just https.

**Comment by kayceesrk** [comment]
Thanks for the comments. Let me confirm that your suggestion does indeed fix the problem and edit the README suitably.

**Comment by kayceesrk** [comment]
I've added a PR #16 to fix README based on your suggestions.

**Comment by kayceesrk** [comment]
I've merged in this PR#16. Thanks @gasche.
